include/aikido/constraint/TSR.hpp

```diff
@@ -112,6 +114,14 @@ class TSR : public Sampleable,
   bool project(const statespace::StateSpace::State* _s,
       statespace::StateSpace::State* _out) const override;
 
+  /// Get the staisfiable tolerance for isSatisfiable.
+  /// \param[out] _out Satisfialble tolerance, double.
+  double getSatisfiableTolerance();
```

Comment:
Change this to `getTestableTolerance()`. Although, I wonder why these set and get methods are necessary. Why do we want to change the tolerance once a TSR is created?